openstack/networking/v2/extensions/vpnaas/siteconnections/requests.go

```diff
@@ -0,0 +1,137 @@
+	// Default is 120.
+	Timeout int `json:"timeout,omitempty"`
+
+	// The dead peer detection (DPD) interval, in seconds.
+	// A valid value is a positive integer.
+	// Default is 30.
+	Interval int `json:"interval,omitempty"`
+}
+
+// CreateOpts contains all the values needed to create a new IPSec site connection
+type CreateOpts struct {
+	// The ID of the IKE policy
+	IKEPolicyID string `json:"ikepolicy_id"`
+
+	// The ID of the VPN Service
+	VPNServiceID string `json:"vpnservice_id"`
+
+	// The ID for the endpoint group that contains private subnets for the local side of the connection.
+	// You must specify this parameter with the peer_ep_group_id parameter unless
+	// in backward- compatible mode where peer_cidrs is provided with a subnet_id for the VPN service.
+	LocalEPGroupID string `json:"local_ep_group_id,omitempty"`
+
+	// The ID of the IPsec policy.
+	IPSecPolicyID string `json:"ipsecpolicy_id"`
+
+	// The peer router identity for authentication.
+	// A valid value is an IPv4 address, IPv6 address, e-mail address, key ID, or FQDN.
+	// Typically, this value matches the peer_address value.
+	PeerID string `json:"peer_id"`
+
+	// The ID of the project
+	TenantID string `json:"tenant_id,omitempty"`
+
+	// The ID for the endpoint group that contains private CIDRs in the form < net_address > / < prefix >
+	// for the peer side of the connection.
+	// You must specify this parameter with the local_ep_group_id parameter unless in backward-compatible mode
+	// where peer_cidrs is provided with a subnet_id for the VPN service.
+	PeerEPGroupID string `json:"peer_ep_group_id,omitempty"`
+
+	// An ID to be used instead of the external IP address for a virtual router used in traffic between instances on different networks in east-west traffic.
+	// Most often, local ID would be domain name, email address, etc.
+	// If this is not configured then the external IP address will be used as the ID.
+	LocalID string `json:"local_id,omitempty"`
+
+	// The human readable name of the connection.
+	// Does not have to be unique.
+	// Default is an empty string
+	Name string `json:"name,omitempty"`
+
+	// The human readable description of the connection.
+	// Does not have to be unique.
+	// Default is an empty string
+	Description string `json:"description,omitempty"`
+
+	// The peer gateway public IPv4 or IPv6 address or FQDN.
+	PeerAddress string `json:"peer_address"`
+
+	// The route mode.
+	// A valid value is static, which is the default.
+	RouteMode string `json:"route_mode,omitempty"`
+
+	// The pre-shared key.
+	// A valid value is any string.
+	PSK string `json:"psk"`
+
+	// Indicates whether this VPN can only respond to connections or both respond to and initiate connections.
+	// A valid value is response-only or bi-directional. Default is bi-directional.
+	Initiator Initiator `json:"initiator,omitempty"`
+
+	// (Deprecated) Unique list of valid peer private CIDRs in the form < net_address > / < prefix > .
```

Comment:
I recommend removing the `(Deprecated)` part here unless you can give further information about the deprecation (ie: when will it be removed and what is the recommended alternative).

I don't see any notes in the Python code mentioning deprecation. It's always possible the API doc is out of date and maybe this field is no longer deprecated?